ams/optimizer/flink-optimizer/pom.xml

```diff
@@ -33,8 +33,9 @@
     <url>https://amoro.netease.com</url>
 
     <properties>
-        <optimizer-flink.version>1.14.6</optimizer-flink.version>
-        <flink.scala.suffix>_2.12</flink.scala.suffix>
+        <flink-optimizer.flink-version>1.18.1</flink-optimizer.flink-version>
+        <flink-optimizer.scala-version>2.12</flink-optimizer.scala-version>
+        <flink-optimizer.scala-version_suffix></flink-optimizer.scala-version_suffix>
```

Comment:
The empty property value may be confusing, how about adding some comment here to tell why?